.github/readme.md

```diff
@@ -47,15 +44,10 @@ use ::windows_app::Microsoft::Windows::System::Power::*;
 use ::windows_app::*;
 
 fn main() -> ::windows::core::Result<()> {
-    bootstrap::initialize()
-        .and_then(|_| {
-            println!(
-                "Remaining charge: {}%",
-                PowerManager::RemainingChargePercent()?
-            );
-            Ok(())
-        })
-        .and_then(|_| bootstrap::uninitialize())
+    bootstrap::initialize()?;
```

Comment:
I take it this sample is out of date?

Reply:
It's current. If you're thinking about winappsdk xcopy deployment, that hasn't shipped yet. 😄